ui/hooks/useTransactionEventFragment.js

```diff
@@ -24,10 +23,7 @@ export const useTransactionEventFragment = () => {
         return;
       }
       if (!fragment) {
-        await createTransactionEventFragment(
-          transaction.id,
-          TransactionMetaMetricsEvent.approved,
-        );
+        await createTransactionEventFragmentWithTxId(transaction.id);
```

Comment:
Since this API is only used, removed second param and renamed it to `createTransactionEventFragmentWithTxId` because the name was so generic and already used within `lib/transaction-metric.ts`

Reply:
I've made a separate comment concerning this, but I'd vote to still bind to your new method but keep the action and background API method name the same to avoid all these changes for a cleaner PR and consistency in the frontend.

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/21330/commits/22e50c9635789db378ed0c925a3e1eebefc28c8b

Reply:
This is fixed in https://github.com/MetaMask/metamask-extension/pull/21330/commits/22e50c9635789db378ed0c925a3e1eebefc28c8b